core/src/main/scala/org/apache/spark/api/python/PythonRunner.scala

```diff
@@ -60,14 +61,20 @@ private[spark] object PythonEvalType {
  */
 private[spark] abstract class BasePythonRunner[IN, OUT](
     funcs: Seq[ChainedPythonFunctions],
-    bufferSize: Int,
-    reuseWorker: Boolean,
     evalType: Int,
-    argOffsets: Array[Array[Int]])
+    argOffsets: Array[Array[Int]],
+    conf: SparkConf)
```

Comment:
Is there any place where this is not `SparkEnv.get.conf`? Just trying to gauge how icky it would be to just hardcode `SparkEnv.get.conf` here.

Reply:
You're right, we could do that. Originally, I thought it was a good idea to pass in the right config, but that's not possible. If we use the config, we must use `SparkEnv.get.conf`. The alternative is to go back to spreading the config logic into every RDD or Exec node to use the conf on the driver and serialize it to make it available on the executors, which is ugly.

I'd prefer just using SparkEnv.get.conf in PythonRunner.